### PR TITLE
Update proto files

### PR DESCRIPTION
**Pull request by ToniRamirezM**
### What does this PR do?

- Updates proto files for executor and prover.
- Enables tx_hash test for executor.

### Reviewers

Main reviewers:

@arnaubennassar 
@fgimenez 
@tclemos

### CODE REVIEW

**Review by fgimenez — APPROVED** [review]

**Review by arnaubennassar — APPROVED** [review]

File: proto/src/proto/zkprover/v1/zk_prover.proto
Comment (tclemos) [inline proto/src/proto/zkprover/v1/zk_prover.proto:16]: Typo

File: proto/src/proto/zkprover/v1/zk_prover.proto
Comment (tclemos) [inline proto/src/proto/zkprover/v1/zk_prover.proto:48]: Not needed

File: proto/src/proto/zkprover/v1/zk_prover.proto
Comment (tclemos) [inline proto/src/proto/zkprover/v1/zk_prover.proto:5]: Is this a definitive change?

**Review by tclemos — APPROVED** [review]
Just small fixes that will not block this PR to be merged but would be good to get fixed before merging

**Pull request merged**
